index.bs

```diff
@@ -1039,6 +1050,31 @@ Note: Invocations of {{MLContext/compute()}} will fail if any of the {{MLContext
 </details>
 </div>
 
+### Errors ### {#api-mlcontext-errors}
+
+<details open algorithm="context-lost">
+<summary>
+When an {{MLContext}} |context| is no longer available to fulfill requests, run the following steps:
+</summary>
+    1. Let |global| be |context|'s [=relevant global object=].
+    1. [=Queue an ML task=] with |global| to run these steps:
+        1. Let |info| be a new {{MLContextLostInfo}}.
+        1. Set |info|.{{MLContextLostInfo/message}} to an [=implementation-defined=] value.
+        1. [=Resolve=] |context|.{{MLContext/[[lost]]}} with |info|.
+</details>
+
+<dl dfn-type=dict-member dfn-for=MLContextLostInfo>
+    : <dfn>message</dfn>
+    :: An [=implementation-defined=] message providing information about the error that occurred.
+</dl>
+
+<div algorithm>
+The <dfn attribute for=MLContext>lost</dfn> getter steps are to return [=this=]'s {{MLContext/[[lost]]}} {{Promise}}.
+</div>
+
+A {{MLContext}} <dfn for=MLContext>is lost</dfn> if its {{MLContext/[[lost]]}} {{Promise}} is settled.
```

Comment:
Can you find a definition to link for "is settled"?

Reply:
7456eff is the best I can do.